variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "user_list" {
```

Comment:
let's give this var a more meaningful name, organization_users/organization_members.